docs/book/stacks-and-components/component-guide/container-registries/gcp.md

```diff
@@ -65,19 +65,6 @@ To figure out the URI for your registry:
   images. If you don't have a repository yet, take a look at the [deployment section](gcp.md#how-to-deploy-it).
 * On the top, click the copy button to copy the full repository URL.
 
-
-#### Infrastructure Deployment
-
-A GCP Container Registry can be deployed directly from the ZenML CLI:
-
-```shell
-zenml container-registry deploy gcp_container_registry --flavor=gcp --provider=gcp ...
```

Comment:
command doesn't work?

Reply:
Nah the whole section was duplicated on the page